.github/workflows/validate-json.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Validate JSON'
+
+on:
+  push:
+    paths:
+      - '**/*.json'
```

Comment:
Sorry I missed this earlier.

According to the [filter pattern cheat sheet](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) `'**.json'` is enough here. I don't understand the patterns well enough to say if this is equivalent to `'**/*.json'` unfortunately

Reply:
In this case it is equivalent.  If you had only wanted to match on a sub directory, you would need the extra `<subdir path>/**/*.json`

Reply:
```suggestion
      - '**.json'
```

Reply:
thanks for the explanation :)